Lib/idlelib/replace.py

```diff
@@ -118,7 +118,7 @@ def _replace_expand(self, m, repl):
         if self.engine.isre():
             try:
                 new = m.expand(repl)
-            except re.error:
+            except re.ReCompileError:
```

Comment:
Do I want the two changes in idlelib?  On the one hand, I prefer the new name.  On the other hand, essentially all idlelib patches are backported and this introduces a potential merge conflict.  On the third hand, I have reviewed these code sections already and do not expect to change them.  So I am not yet requesting a change.

Reply:
Removed from the changes.